package.json

```diff
@@ -12,6 +12,9 @@
     "@pinia/nuxt": "^0.4.6",
     "@vueuse/core": "^9.2.0",
     "@vueuse/nuxt": "^9.2.0",
+    "cartonne": "^2.2.0",
+    "dag-jose": "^4.0.0",
```

Comment:
@Nidz-Web3 About these two new packages - I'll try to replicate the error that you got that required you to install these packages. If I can't reproduce the error, then let's move the packages to devDependencies, because it doesn't look like they are needed in production 🙂